### PR TITLE
Added shanks

### PR DESCRIPTION
**Pull request by ghost**
:cl: Thefastfoodguy
add: Shanks
code: Reorganized makeshift armor and trayshield code
/:cl:

### CODE REVIEW

**Comment by PopNotes** [comment]
>stungloves

NO

File: hippiestation/code/game/objects/items/ducttape.dm [hunk 1]
```diff
@@ -75,6 +75,17 @@
 		if(replace)
 			user.put_in_hands(new_item)
 		playsound(user, 'hippiestation/sound/misc/ducttape1.ogg', 50, 1)
+	if(istype(W, /obj/item/shard) && !istype(W, /obj/item/shard/shank))
```
Comment (KayeArray) [inline hippiestation/code/game/objects/items/ducttape.dm:78]: try: /obj/item/shard/ to get rid of the other !istype

**Comment by tortellinitony** [comment]
People don't want stun gloves

**Comment by DinduStuffin** [comment]
yeah here comes more /vg/ shilling

/vg/ has some neat shit you could try though, like crossbows, swords, wheelchair cannons, etc.
Why not bring those over? I'd love to see people running around with crossbows on some medieval shit.

**Comment by KayeArray** [comment]
add ghetto chainmail

**Comment by Mlgdatboiwew** [comment]
+1 I want this sheeet

**Comment by PopNotes** [comment]
until stungloves are removed, it's a -1 from me

**Comment by DinduStuffin** [comment]
Same as popnotes. Stungloves are super OP imo.

**Comment by ghost** [comment]
stun gloves begone

**Comment by ghost** [comment]
build this

**Comment by ghost** [comment]
wtf it compiles fine locally

**Review by j-mie — APPROVED** [review]

**Pull request merged**